src/navigation/task/components/utils.ts

```diff
@@ -29,3 +29,19 @@ export const getName = task => {
     })
     .join(' ');
 };
+
+export const getPackagesSummary = task => {
```

Comment:
I think I already saw this function in another file I was reviewing :thinking: 
If that's the case, try to re-use one single function!